doc/pages/administrator/030-installation_kubernetes.md

```diff
@@ -17,87 +17,11 @@ helm install astarte-operator astarte/astarte-operator
 ```
 
 This command will take care of installing all needed components for the Operator to run. This
-includes all the RBAC roles, Custom Resource Definitions, and the Operator itself.
+includes all the RBAC roles, Custom Resource Definitions, Webhooks, and the Operator itself.
 
 You can use the `--version` switch to specify a version to install. When not specified, the latest
 stable version will be installed instead.
 
-## Upgrading the Operator
-
-The procedure for upgrading the Operator depends on the version of the Operator you want to
-upgrade from.
```

Comment:
Instead of completely removing the section, leave it and link to the newly created section